definitions/ext-service/definition.yml

```diff
@@ -8,18 +8,59 @@ synthesis:
       encodeIdentifierInGUID: true
       conditions:
         - attribute: service.name
+
     # telemetry with service_name attribute
     - identifier: service_name
       name: service_name
       encodeIdentifierInGUID: true
       conditions:
         - attribute: service_name
+
     # telemetry with serviceName attribute
     - identifier: serviceName
       name: serviceName
       encodeIdentifierInGUID: true
       conditions:
         - attribute: serviceName
+
+    # telemetry from opentelemetry provider
+    - identifier: service.name
+      name: service.name
+      encodeIdentifierInGUID: true
+      conditions:
+        - attribute: telemetry.sdk.name
+          value: opentelemetry
+      tags:
+        telemetry.sdk.name:
+          entityTagName: instrumentation.provider
+        telemetry.sdk.language:
+          entityTagName: language
+        telemetry.sdk.version:
+
+    # telemetry from kamon-agent provider
+    - identifier: service.name
+      name: service.name
+      encodeIdentifierInGUID: true
+      conditions:
+        - attribute: instrumentation.provider
+          value: kamon-agent
+      tags:
+        instrumentation.provider:
```

Comment:
This tag is already added by default I think ( @srvaroa)

Reply:
it should be, yes